scripts/automation/create_release_tag.py

```diff
@@ -0,0 +1,446 @@
+
+
+def get_history_note(wheel_url: str, version: str) -> str:
+    """Download wheel package and extract HISTORY.rst to find version notes"""
+    try:
+        # Download wheel file
+        response = requests.get(wheel_url)
+        response.raise_for_status()
+
+        with tempfile.TemporaryFile() as temp_file:
+            temp_file.write(response.content)
+            temp_file.seek(0)
+
+            # Open wheel as zip
+            with zipfile.ZipFile(temp_file, 'r') as wheel:
+                # Find DESCRIPTION.rst file
+                history_files = [f for f in wheel.namelist() if f.endswith('DESCRIPTION.rst')]
+                if not history_files:
+                    return "No history notes found"
+
+                history_content = wheel.read(history_files[0]).decode('utf-8')
+
+                # Match any line starting with the version number
+                version_pattern = rf"^{re.escape(version)}.*?\n(?:[-=+~]+\n)?(.*?)(?=^[\d.]+[a-z0-9].*?(?:\n[-=+~]+)?|\Z)"
+                match = re.search(version_pattern, history_content, re.DOTALL | re.MULTILINE)
+
+                if match:
+                    return match.group(1).strip()
+
+                return "No history notes found for this version"
+
+    except Exception as e:
+        print(f"Error getting history notes: {e}")
+        return "No history notes found for this version"
+
+
+def get_history_note_from_source(version: str, extension_name: str) -> str:
+    """Get history notes from source code HISTORY.rst"""
+    try:
+        history_path = f"src/{extension_name}/HISTORY.rst"
+        if not os.path.exists(history_path):
+            return "No history notes found"
+
+        with open(history_path, 'r', encoding='utf-8') as f:
+            history_content = f.read()
+
+        # Match any line starting with the version number
+        version_pattern = rf"^{re.escape(version)}.*?\n(?:[-=+~]+\n)?(.*?)(?=^[\d.]+[a-z0-9].*?(?:\n[-=+~]+)?|\Z)"
+        match = re.search(version_pattern, history_content, re.DOTALL | re.MULTILINE)
+
+        if match:
+            return match.group(1).strip()
+
+        return "No history notes found in source code"
+
+    except Exception as e:
+        print(f"Error reading history from source: {e}")
+        return "No history notes found for this version"
+
+
+def main():
+    try:
+        # Get added lines from git diff
+        added_lines = get_file_diff()
+        if not added_lines:
+            print("No changes found in index.json")
+            return
+
+        # Parse filenames from added lines
+        filenames = parse_filenames(added_lines)
+        commit_sha = subprocess.check_output(
+            ["git", "rev-parse", "HEAD"],
+            text=True
+        ).strip()
+        if not filenames:
+            print("No filenames found in changes")
+            shas = parse_sha256_digest(added_lines)
+            if not shas:
```

Comment:
If someone only modify the SHA, we need to update the tag, release, and re-upload the wheel package.